github/create-pull-request.ps1

```diff
@@ -87,8 +84,9 @@ Git-CommitAllChanges -Message $CommitMessage
 Write-Host "Push branch: $BranchName"
 Git-PushBranch -Name $BranchName -Force $true
 
-$gitHubApi = Get-GitHubApi -AccountName $RepositoryOwner -ProjectName $RepositoryName -AccessToken $AccessToken
-$pullRequest = $gitHubApi.GetPullRequest($BranchName, $RepositoryOwner)
+$gitHubApi = Get-GitHubApi -RepositoryFullName $RepositoryFullName -AccessToken $AccessToken
+$repositoryOwner = $RepositoryFullName.Split('/')[0]
+$pullRequest = $gitHubApi.GetPullRequest($BranchName, $repositoryOwner)
```

Comment:
Why this method requires `repositoryOwner`? Can it be consumed from class directly? I think it will be easier

Reply:
Thanks!